lib/fog/compute/kubevirt/models/vms.rb

```diff
@@ -24,6 +26,99 @@ def all(filters = {})
         def get(name)
           new service.get_vm(name)
         end
+
+        def create(args = {})
+          vm_name = args.fetch(:vm_name)
```

Comment:
where does the list of arguments being retrieved from ?  meaning where do we define properties named 'memory_size' and 'cpus' ? Couldn't find such on any repo

Reply:
`create` method is called by the user of this library. It is not here in the repo.